editor/project_converter_3_to_4.cpp

```diff
@@ -700,6 +703,7 @@ static const char *csharp_function_renames[][2] = {
 	{ "GetCamera", "GetCamera3d" }, // Viewport -> this is also convertable to getCamera2d, broke GLTFNode
 	{ "GetCancel", "GetCancelButton" }, // ConfirmationDialog
 	{ "GetCaption", "_GetCaption" }, // AnimationNode
+	{ "GetCaretBlinkSpeed", "GetCaretBlinkInterval" }, // TextEdit, LineEdit
```

Comment:
This wasn't the `3.x` name of these methods. It's `Cursor{Set,Get}BlinkSpeed` that should be changed.